src/main/java/frc/robot/commands/DelayCommand.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.commands;
```

Comment:
See simple one shot command in my sample branch. That, combined with anonymous inner classes,eliminates tedious classes like this one